.github/workflows/publish_client_linux_release_candidate.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     timeout-minutes: 15
     env:
       RELEASES_REPOSITORY: ${{ secrets.RELEASES_REPOSITORY }}
-      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      RELEASES_DEPLOY_TOKEN: ${{ secrets.RELEASES_DEPLOY_TOKEN }}
```

Comment:
github security recommends using deploy keys, not access keys, for this sort of thing. idk if we can tag though (we'll see)